src/test/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
+            </Pattern>
+        </layout>
+    </appender>
+
+    <root level="error">
+        <appender-ref ref="CONSOLE"/>
+    </root>
+</configuration>
```

Comment:
Logback does not seem to be configured by Maven? Instead there is slf4j-simple. Then this file isn't needed. It could still make sense to use logback in both tests and in the actual release, to also write the logs to file for example, but then the dependencies need to be adjusted.